publish2wiki.py

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env python
+
```

Comment:
I'd like to ask if you could add a comment here with the following:

 * How to install the necessary dependencies
 * How to make 'data.txt' and where it would come from
 * How to use this script? Why you would use this script?

Reply:
I want to schedule a ./test-stats run on centos openshift.
1. Dependency are installed as a part of Job definition.
2. data.txt comes from ./test-stats output
3. I just decided to put it here. If you do not want I can put it somewhere else.

Reply:
Sorry, I wasn't clear. Could you add this as a comment *in the code*  ... so that everyone who looks at it understands immediately what it's for, and how to use it.

Reply:
Updated

Reply:
Could you add:

 * How to install the necessary dependencies